docs/conf.py

```diff
@@ -18,7 +18,7 @@
 html_context = {
   'version' : '3.1',
   'full_version' : '3.1.0-RC7',
-  'maven_plugin_version' : '3.1.0-RC6',
+  'maven_plugin_version' : '3.1.0-RC7',
```

Comment:
Maven plugin has no RC7 for now.

http://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs-maven-plugin/